frontend/src/pages/my-projects/components/project-card/project-card.tsx

```diff
@@ -13,25 +21,65 @@ type Properties = {
 };
 
 const ProjectCard: React.FC<Properties> = ({ project }: Properties) => {
+  const dispatch = useAppDispatch();
   const { id, name, avatarUrl, category } = project;
 
+  const {
+    isOpenModal: isConfigurateProject,
+    handleModalClose: handleProjectModalClose,
+    handleModalOpen: handleProjectModalOpen,
+  } = useModal();
+
+  const handleSubmitUpdateProject = useCallback(
+    (payload: ProjectRequestDto & ProjectUploadImageDto): void => {
+      void dispatch(projectActions.updateProject({ projectId: id, payload }))
+        .unwrap()
+        .then(handleProjectModalClose);
+    },
+    [dispatch, handleProjectModalClose, id],
+  );
+
   return (
-    <div className={styles['card']}>
-      <Image className={styles['card-image']} src={avatarUrl ?? img} alt="" />
-      <div className={styles['card-description']}>
-        <Link
-          to={configureString<ValueOf<typeof AppRoute>>(
-            AppRoute.PROJECTS_$PROJECT_ID_SITES,
-            {
-              projectId: id,
-            },
-          )}
-        >
-          <h2>{name}</h2>
-          <p>{category}</p>
-        </Link>
-      </div>
-    </div>
+    <>
+      {isConfigurateProject &&
+        createPortal(
```

Comment:
```suggestion
```
one modal should be placed in `frontend/src/pages/my-projects/my-projects.tsx` for both, edit and creadtion